Classes/Error/FusionExceptionView.php

```diff
@@ -0,0 +1,172 @@
+<?php
+namespace Sitegeist\MoveAlong\Error;
+
+use Neos\Flow\Annotations as Flow;
+use Neos\Flow\Core\Bootstrap;
+use Neos\Flow\Mvc\View\ViewInterface;
+use Neos\Flow\Mvc\View\AbstractView;
+use Neos\Neos\Domain\Service\FusionService;
+use Neos\Fusion\Core\Runtime as FusionRuntime;
+use Neos\Neos\Domain\Repository\SiteRepository;
+use Neos\Neos\Domain\Repository\DomainRepository;
+use Neos\Neos\Domain\Service\ContentContextFactory;
+use Neos\ContentRepository\Domain\Model\NodeInterface;
+use Neos\Flow\I18n\Locale;
+use Neos\Flow\I18n\Service;
+use Neos\Flow\Security\Context;
+use Neos\Flow\ObjectManagement\ObjectManagerInterface;
+use Neos\Flow\Http\Response;
+use Neos\Flow\Mvc\ActionRequest;
+use Neos\Flow\Mvc\Routing\UriBuilder;
+use Neos\Flow\Mvc\Controller\ControllerContext;
+use Neos\Flow\Mvc\Controller\Arguments;
+
+class FusionExceptionView extends AbstractView implements ViewInterface
+{
+
+    /**
+     * @Flow\Inject
+     * @var Bootstrap
+     */
+    protected $bootstrap;
+
+    /**
+     * @var ObjectManagerInterface
+     * @Flow\Inject
+     */
+    protected $objectManager;
+
+    /**
+     * @Flow\Inject
+     * @var Service
+     */
+    protected $i18nService;
+
+    /**
+     * @var FusionService
+     * @Flow\Inject
+     */
+    protected $fusionService;
+
+    /**
+     * @var FusionRuntime
+     */
+    protected $fusionRuntime;
+
+    /**
+     * @var SiteRepository
+     * @Flow\Inject
+     */
+    protected $siteRepository;
+
+    /**
+     * @var DomainRepository
+     * @Flow\Inject
+     */
+    protected $domainRepository;
+
+    /**
+     * @var ContentContextFactory
+     * @Flow\Inject
+     */
+    protected $contentContextFactory;
+
+    /**
+     * @return string
+     * @throws \Neos\Flow\Security\Exception
+     * @throws \Neos\Neos\Domain\Exception
+     */
+    public function render()
+    {
+        $domain = $this->domainRepository->findOneByActiveRequest();
+
+        if ($domain) {
+            $site = $domain->getSite();
+        } else {
+            $site = $this->siteRepository->findDefault();
+        }
+
+        $httpRequest = $this->bootstrap->getActiveRequestHandler()->getHttpRequest();
+        $request = new ActionRequest($httpRequest);
+        $request->setControllerPackageKey('Neos.Neos');
+        $request->setFormat('html');
+        $uriBuilder = new UriBuilder();
+        $uriBuilder->setRequest($request);
+        $controllerContext = new ControllerContext(
```

Comment:
The `AbstractView` has a protected `controllerContext` though. Any reason to build a new one here?

Using the already existing context sure solves your following problem:
> Also i am not sure wether there is a way to get access to the http-request so it is not needed to fake the request for this.

Reply:
Not specific, i found out how to access the current request via bootstrap later. So maybe i did not notice I might eliminate this aswell. Will try it out

Reply:
Wait, i ran into trouble when i rendered 410 status messages because in that case the mvc mechanism is skipped entirely and no proper controllerContext exists. That is why I had to create an entirely fresh context

Reply:
So, the controllerContext is `null` then? That doesn't seem right... We should at least open an issue over at neos-development-collection (Or is it Flow?)

Reply:
Okay, I'd say we leave it as is then.